lib/i18n/tasks/translation.rb

```diff
@@ -22,7 +22,7 @@ def translate_forest(forest, from:, backend: :google)
       when :yandex
         Translators::YandexTranslator.new(self).translate_forest(forest, from)
       else
-        fail CommandError, "invalid backend: #{backend}"
+        fail CommandError, backend.present? ? "invalid backend: #{backend}" : 'no backend specified'
```

Comment:
AFAICT this isn't translated because it should never happen - the option parser should fail before this line is reached